components/CallToAction/StyledCallToAction.tsx

```diff
@@ -0,0 +1,17 @@
+import styled from '@emotion/styled'
+
+export const StyledCallToAction = styled.div`
+    .MuiSpeedDial-fab {
+        background-color: var(--accent-color-2);
+        color: var(--off-brand-dark);
+        &:hover {
+            background-color: var(--accent-color-2);
+        }
+    }
+    #SpeedDialCTA-actions {
```

Comment:
Doesnt look like these styles are working. you might need to add specific classes to the MUI components and style them in the `styles/overrides.css` file if you cant do it locally.

Reply:
This works for me on chrome safari and mozilla .